tinker/faculty_bios/faculty_bio_controller.py

```diff
@@ -135,7 +135,8 @@ def _iterate_child_xml(self, child, author, csv=False):
                     'path': 'https://www.bethel.edu' + child.find('path').text or "",
                     'schools': school_array,
                     'last-name': child.find('.//last').text or None,
-                    'deactivated': child.find('.//deactivate').text or None
+                    'deactivated': child.find('.//deactivate').text or None,
+                    'courseleaf-user': child.find('.//courseleaf-user').text or None
```

Comment:
make sure you are handling this, if its None, in the template

Reply:
changed to ... or "Yes"